packages/web-components/src/components/ic-breadcrumb-group/ic-breadcrumb-group.tsx

```diff
@@ -128,7 +136,14 @@ export class BreadcrumbGroup {
     }
   };
 
-  private renderCollapsedBreadcrumb = () => {
+  private clickHandler = () => {
+    this.handleHiddenCollapsedBreadcrumbs(this.collapsedBreadcrumbWrapper);
+  };
+
+  private renderCollapsedBreadcrumb = (action: "add" | "remove") => {
```

Comment:
when run with `remove` passed as param, won't this still create the content in the DOM?